rasa/core/training/interactive.py

```diff
@@ -1477,7 +1477,12 @@ async def record_messages(
 
         domain_intents = domain.get("intents", []) if domain is not None else []
 
-        intents = [next(iter(i)) for i in domain_intents]
+        intents = [
```

Comment:
Could we add a brief comment to clarify that the intent dict is only possible when adding `use_entities` or `ignore_entities` properties to an intent in the domain?